src/app/Http/Controllers/Admin/MenuItemCrudController.php

```diff
@@ -53,6 +53,7 @@ public function setup()
                 'label' => 'Type',
                 'type' => 'page_or_link',
                 'page_model' => '\Backpack\PageManager\app\Models\Page',
+                'view_namespace' => file_exists(resource_path('views/vendor/backpack/crud/fields/page_or_link.blade.php')) ? null : 'pagemanager::fields',
```

Comment:
We have used this code for the other PageManager field; `select_page_template`, if the field is published; no namespace – fallback to developers code, otherwise, `'pagemanager::fields'` namespace.